client/tests/TestPlanReportStatusDialog.test.jsx

```diff
@@ -14,8 +14,51 @@ import TestPlanReportStatusDialog from '../components/TestPlanReportStatusDialog
 import { TEST_PLAN_REPORT_STATUS_DIALOG_MOCK_DATA } from './__mocks__/GraphQLMocks';
 // eslint-disable-next-line jest/no-mocks-import
 import { mockedTestPlanVersion } from './__mocks__/GraphQLMocks/TestPlanReportStatusDialogMock';
+import { TEST_PLAN_RUN_REPORTS_INITIATED_BY_AUTOMATION } from '../components/AddTestToQueueWithConfirmation/queries';
+import { ME_QUERY } from '../components/App/queries';
 
-const setup = (props, mocks = []) => {
+const mocks = [
+    {
```

Comment:
I think this could be added to `TestPlanReportStatusDialogMock.js` instead. By updating `GraphQLMocks/index#TEST_PLAN_REPORT_STATUS_DIALOG_MOCK_DATA` with the `TEST_PLAN_RUN_REPORTS_INITIATED_BY_AUTOMATION` param and then adding this request to the result in `TestPlanReportStatusDialogMock.js`.

What do you think of doing it that way?

Reply:
Great point! I must've written this inline mock without realizing there was already a bigger mock in place. Thanks for the suggestion, addressed in [feaffa0](https://github.com/w3c/aria-at-app/pull/905/commits/feaffa08af73b3c3aef98c5b9420881ee6716a33)